spec/models/ivc_champva_forms_spec.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe IvcChampvaForm, type: :model do
```

Comment:
What about some non-happy path tests like if the email is not present or unique?

Reply:
Ah thanks, I knew I was forgetting something